client/lib/importer/test/api-interaction.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import Dispatcher from 'dispatcher';
+import nock from 'nock';
+import partial from 'lodash/function/partial';
+
+import { fetchState } from '../actions';
+import { actionTypes } from '../constants';
+import store from '../store';
+
+const fetchTestState = partial( fetchState, 0 );
+const hydratedState = () => store.get().getIn( [ 'api', 'isHydrated' ] );
+const resetStore = () => Dispatcher.handleViewAction( { type: actionTypes.RESET_STORE } );
+
+const queuePayload = payload =>
+	nock( 'https://public-api.wordpress.com:443' )
+		.get( '/rest/v1.1/sites/0/imports/' )
+		.replyWithFile( 200, `${ __dirname }/api-payloads/${ payload }.json` );
```

Comment:
The responses feel small enough that they can be left inline with the test. If a test object is complicated enough to need moving to another file I would suggest using a Factory pattern to generate test data instead, using a library like https://github.com/rosiejs/rosie. The latter is probably out of scope for this PR. :)